Userland/Libraries/LibJS/Runtime/ECMAScriptFunctionObject.cpp

```diff
@@ -558,6 +558,44 @@ void ECMAScriptFunctionObject::visit_edges(Visitor& visitor)
         });
 }
 
+// 10.2.5 MakeConstructor ( F [ , writablePrototype [ , prototype ] ] ), https://tc39.es/ecma262/#sec-makeconstructor
+void ECMAScriptFunctionObject::make_constructor(bool writable_prototype, Object* prototype)
```

Comment:
the signature here should be `make_constructor(GCPtr<Object> prototype = nullptr)` instead of passing two arguments, one of which is a raw pointer.